### DISCUSSION TITLE
Update a plugin for the new api

### DISCUSSION THREAD

**Discussion by jugandomiguel** [Pryaxis/TShock#2663] [Q&A]
Hi, I want to change this plugin for run in the newest version https://tshock.co/xf/index.php?resources/teleportrequest.173/

**Comment by Arthri** [comment]
Hello! Do you have prior experience with C#?

**Comment by jugandomiguel** [comment]
Nop

**Comment by MaksDragon** [comment]
are you trying -ignoreversion ?
